django/db/backends/base/base.py

```diff
@@ -267,6 +267,8 @@ def rollback(self):
         self._rollback()
         # A successful rollback means that the database connection works.
         self.errors_occurred = False
+        # We just rolled back, so a rollback is no longer needed
```

Comment:
I try to phrase comments without "we", e.g. "After a rollback, a rollback is no longer needed." although it seems like maybe a comment doesn't provide much benefit?